### PR TITLE
Added beta motor neurons and subclasses #739

### LINKED ISSUE #739: Add beta motor neuron
https://en.wikipedia.org/wiki/Beta_motor_neuron

### PR DESCRIPTION
**Pull request by shawntanzk**
Fixes #739

### CODE REVIEW

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -27032,6 +27034,27 @@ EquivalentClasses(obo:CL_3000004 ObjectIntersectionOf(obo:CL_0000111 ObjectSomeV
 SubClassOf(obo:CL_3000004 obo:CL_0000101)
 SubClassOf(obo:CL_3000004 obo:CL_0000111)
 
+# Class: obo:CL_4023000 (beta motor neuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:25346659") obo:IAO_0000115 obo:CL_4023000 "A motor neuron that innervates both intrafusal and extrafusal muscle fibers. Not well characterised due to its low abundancy. They control both muscle contraction and responsiveness of the sensory feedback from muscle spindles.")
+AnnotationAssertion(oboInOwl:created_by obo:CL_4023000 <http://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(rdfs:label obo:CL_4023000 "beta motor neuron")
+SubClassOf(obo:CL_4023000 obo:CL_0008039)
```
Comment (dosumis) [inline src/ontology/cl-edit.owl:27042]: Looks good.  Add:
```
subClassOf: synpased_to some 'extrafusal muscle fiber'
subClassOf: synpased_to some 'intrafusal muscle fiber'
```

For text def - remove 'not well characterised' as this is likely to change, but leave mention of 'low abundance'

File: src/ontology/cl-edit.owl [hunk 2]
```diff
@@ -27032,6 +27034,27 @@ EquivalentClasses(obo:CL_3000004 ObjectIntersectionOf(obo:CL_0000111 ObjectSomeV
...
+AnnotationAssertion(rdfs:label obo:CL_4023001 "static beta motor neuron")
+SubClassOf(obo:CL_4023001 obo:CL_4023000)
+
+# Class: obo:CL_4023002 (dynamic beta motor neuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:25346659") obo:IAO_0000115 obo:CL_4023002 "A beta motor neuron that innervates the nuclear bag fibers of muscle spindles.")
+AnnotationAssertion(oboInOwl:created_by obo:CL_4023002 <http://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(rdfs:label obo:CL_4023002 "dynamic beta motor neuron")
+SubClassOf(obo:CL_4023002 obo:CL_4023000)
+
```
Comment (dosumis) [inline src/ontology/cl-edit.owl:27057]: Please make tickets for spindle component 'nuclear chain fiber' and 'nuclear bag fiber'

File: src/ontology/cl-edit.owl [hunk 3]
```diff
@@ -21969,16 +21981,6 @@ AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0009019 "cortical nephrogeni
...
-# Class: obo:CL_0009018 (lymphocyte of large intestine lamina propria)
-
-AnnotationAssertion(obo:IAO_0000115 obo:CL_0009018 "A lymphocyte that resides in the lamina propria of the large intestine.")
-AnnotationAssertion(<http://purl.obolibrary.org/obo/pato#location_grouping> obo:CL_0009018 "")
-AnnotationAssertion(oboInOwl:created_by obo:CL_0009018 <http://orcid.org/0000-0002-2825-0621>)
-AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0009018 "https://orcid.org/0000-0003-4183-8865")
-AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0009018 "lamina propria lymphocyte of large intestine")
-AnnotationAssertion(rdfs:label obo:CL_0009018 "lymphocyte of large intestine lamina propria")
-EquivalentClasses(obo:CL_0009018 ObjectIntersectionOf(obo:CL_0000542 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0011189)))
-
```
Comment (dosumis) [inline src/ontology/cl-edit.owl:21981]: Something odd going on here with diffs.  Ignore for now as may disappear.

**Review by dosumis — CHANGES_REQUESTED** [review]
Changes requested in comment on 'beta motor neuron' required for review.

File: src/ontology/cl-edit.owl (hunk 3 above)
Comment (shawntanzk) [reply src/ontology/cl-edit.owl:21981]: Yeah, I noticed it too, but it seems to add it back too? So I guess it's fine?

**Review by dosumis — APPROVED** [review]

File: src/ontology/cl-edit.owl [hunk 4]
```diff
@@ -2532,6 +2533,10 @@ AnnotationAssertion(rdfs:label obo:IAO_0000700 "preferred_root")
 
 AnnotationAssertion(rdfs:label obo:IAO_0100001 "term replaced by"^^xsd:string)
 
+# Annotation Property: <http://purl.obolibrary.org/obo/cl#added_for_HCA> (<http://purl.obolibrary.org/obo/cl#added_for_HCA>)
```
Comment (nicolevasilevsky) [inline src/ontology/cl-edit.owl:2536]: not sure what this is? should this be removed?

Comment (nicolevasilevsky) [reply src/ontology/cl-edit.owl:2536]: (this came in when I fixed the conflict, btw)

Comment (dosumis) [reply src/ontology/cl-edit.owl:2536]: It's fine.  Added by Paola in another PR & fixed by me.

**Pull request merged**